test/raise_on_unpermitted_params_test.rb

```diff
@@ -30,4 +30,15 @@ def teardown
       params.permit(:book => [:pages])
     end
   end
-end
\ No newline at end of file
+  
+  test "not raise on params included in NEVER_UNPERMITTED_PARAMS" do
+    key = ActionController::Parameters::NEVER_UNPERMITTED_PARAMS[0]
+    params = ActionController::Parameters.new({
+      key => "Turnips"
```

Comment:
It's better to test each entries of the NEVER_UNPERMITTED_PARAMS array. isn't it?